gettsim/data/elterngeld.yaml

```diff
@@ -34,3 +45,89 @@ elgzwst:
     2013-01-01:
       value: 1201
       note:
+elgmin:
+  name:
+    de: Mindestbetrag des Elterngeldes
+    en:
+  description:
+    de: § 2 (4) BEEG
+    en:
+  values:
+    2007-01-01:
+      value: 300
+      note:
+elg_perc_correct:
+  name:
+    de: Percentage correction for lower or high incomes
+    en:
+  description:
+    de: § 2 (2) BEEG
+    en:
+  values:
+    2013-01-01:
+      value: 0.001
+      note:
+elg_perc_correct_step:
+  name:
+    de: Income steps for percentage correction
+    en:
+  description:
+    de: § 2 (2) BEEG
+    en:
+  values:
+    2013-01-01:
+      value: 2
+      note:
+elg_geschw_bonus_share:
+  name:
+    de: Geschwisterbonus als prozentualer Aufschlag
+    en: Sibling Bonus, percentage top-up
+  description:
+    de: §2a (1) BEEG
+    en:
+  values:
+    2007-01-01:
+      value: 0
+      note: implizit, exisitiert vorher nicht
+    2012-09-18:
+      value: 0.1
+      note: Artikel 1 G. v. 10.09.2012 BGBl. I S. 1878
+elg_geschw_bonus_min:
+  name:
+    de: Minimaler Geschwisterbonus als Euro-Betrag
+    en: Sibling Bonus, minimal amount
+  description:
+    de: §2a (1) BEEG
+    en:
+  values:
+    2007-01-01:
+      value: 0
+      note: implizit, exisitiert vorher nicht
+    2012-09-18:
+      value: 75
+      note: Artikel 1 G. v. 10.09.2012 BGBl. I S. 1878
+elg_geschw_bonus_max:
+  name:
+    de: Maximaler Geschwisterbonus als Euro-Betrag
+    en: Sibling Bonus, maximal amount
+  description:
+    de: §2a (1) BEEG
+    en:
+  values:
+    2007-01-01:
+      value: 0
+      note: implizit, exisitiert vorher nicht
+    2012-09-18:
+      value: 180
+      note: Artikel 1 G. v. 10.09.2012 BGBl. I S. 1878
+soz_vers_pausch:
+  name:
+    de: Sozialversicherungspauschale zur Berechnung des Nettoeinkommens
+    en:
+  description:
+    de: § 153 (1) Nr. 1 SGB III
+    en:
```

Comment:
It's §2f BEEG and it's 0.21. Could you at least mention the individual components (9 + 10 + 2) in the note?